src/status_im/subs/wallet/collectibles.cljs

```diff
@@ -38,7 +38,22 @@
      (assoc last-collectible :preview-url (preview-url (:collectible-data last-collectible))))))
 
 (re-frame/reg-sub
- :wallet/last-collectible-chain-id
+ :wallet/last-collectible-details-chain-id
  :<- [:wallet/last-collectible-details]
  (fn [collectible]
    (get-in collectible [:id :contract-id :chain-id])))
+
+(re-frame/reg-sub
+ :wallet/last-collectible-details-traits
+ :<- [:wallet/last-collectible-details]
+ (fn [collectible]
+   (get-in collectible [:collectible-data :traits])))
+
+(re-frame/reg-sub
+ :wallet/last-collectible-details-owner
+ :<- [:wallet/last-collectible-details]
+ :<- [:wallet]
+ (fn [[collectible wallet]]
+   (let [address (:address (first (:ownership collectible)))
```

Comment:
The guidelines recommend testing layer-3 subs https://github.com/status-im/status-mobile/blob/563f1c588d1cdac31629da009a1c894133038d2d/doc/new-guidelines.md#registering-layer-3-subscriptions

I see other layer-3 subs in this file are not tested either.

Reply:
Done @ilmotta